.github/actions/templates/avm-getWorkflowInput/action.yml

```diff
@@ -79,7 +79,7 @@ runs:
         # Otherwise retrieve default values
         else {
           # Load used functions
-          . (Join-Path $env:GITHUB_WORKSPACE 'avm' 'utilities' 'pipelines' 'sharedScripts' 'Get-GitHubWorkflowDefaultInput.ps1')
+          . (Join-Path $env:GITHUB_WORKSPACE 'utilities' 'pipelines' 'sharedScripts' 'Get-GitHubWorkflowDefaultInput.ps1')
```

Comment:
Placeholder comment to disable actions before merge

Reply:
@eriqua, approved